### PR TITLE
[13.0][IMP] sale_order_type: Add analytic account and active fields to order type

### PR DESCRIPTION
**Pull request by sergio-teruel**
Cherry pick from https://github.com/OCA/sale-workflow/commit/dae324fc40ab86b098aa81f69de52e46f4a72c62

cc @Tecnativa

ping @carlosdauden @chienandalu

### CODE REVIEW

**Comment by sergio-teruel** [comment]
Unrelated travis fail

**Review by carlosdauden — APPROVED** [review]

**Review by chienandalu — APPROVED** [review]

File: sale_order_type/models/sale_order_type.py [hunk 1]
```diff
@@ -59,3 +59,9 @@ def default_picking_policy(self):
         ondelete="restrict",
         check_company=True,
     )
+    analytic_account_id = fields.Many2one(
+        comodel_name="account.analytic.account",
+        string="Analytic account",
+        check_company=True,
+    )
+    active = fields.Boolean(default=True)
```
Comment (rousseldenis) [inline sale_order_type/models/sale_order_type.py:72]: @sergio-teruel Could you put this also in commit description ?

**Comment by sergio-teruel** [comment]
@rousseldenis Done!!

**Review by Tardo — APPROVED** [review]
Code review

**Pull request merged**